app/components/DynamicForm/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { Form as F, FormInstance } from 'antd';
+import styled from 'styled-components';
+import FormInput from '@components/FormInput';
+import { StepperLoginFormInputInterface } from '@app/interfaces/common';
+
+const FormContainer = styled.div`
+  display: flex;
+  flex: 1 1 0;
+  align-self: center;
+  padding: 1rem;
+`;
+
+const Form = styled(F)`
+  width: 20rem;
+`;
+interface DynamicStepFormProps {
+  formInput: StepperLoginFormInputInterface;
+  handleFormChange: (value: FormInstance<any>) => void;
+  handleSubmit: (value: any) => void;
+}
+
+const DynamicStepForm = ({ formInput, handleSubmit, handleFormChange }: DynamicStepFormProps) => {
+  const [form] = Form.useForm();
+
+  useEffect(() => {
+    handleFormChange(form);
```

Comment:
What are we doing here?

Reply:
so we are sending that form instance from child to parent